app/actions/ControlActions.js

```diff
@@ -654,3 +657,13 @@ export const validateAddress = address => async (dispatch, getState) => {
     throw error;
   }
 };
+
+export const validateWalletAddress = address => async (dispatch, getState) => {
+  const { network } = getState().grpc;
+  const validationErr = isValidAddress(address.address, network);
+  if (validationErr) { return { isValid: false, error: validationErr, getIsValid () { false; } }; }
+  dispatch({ type: VALIDATEADDRESS_ATTEMPT });
+  wallet.validateAddress(sel.walletService(getState()), address.address)
+    .then(response => { dispatch({ response, type: VALIDATEADDRESS_SUCCESS });})
```

Comment:
Minor nit. 

Seems to me the only difference between `validateAddress` and `validateWalletAddress` is the try...catch and the dispatch of VALIDATEADDRESS_SUCCESS. Maybe we could move the VALIDATEADDRESS_SUCCESSS dispatch into validateAddress and get rid of validateWalletAddress altogether? Or make validateWalletAddress use validateAddress and dispatch the VALIDATEADDRESS_SUCCESS in case of no errors.

This would be mainly to avoid the duplicate code...

Reply:
Yah I did it sloppy on this first pass due to trying to mutlicase the existing one.  I'll fix before merging this PR.